import-scripts/oncokb-annotator.sh

```diff
@@ -297,10 +282,21 @@ FLOCK_FILEPATH="/data/portal-cron/cron-lock/oncokb-annotator.sh"
     if [ $ONCOKB_ANNOTATION_SUCCESS -eq 1 ] ; then
         echo $(date)
         echo "Beginning README generation..."
-        $PYTHON3_BINARY $README_SCRIPT -o $ONCOKB_README_FILE -u $ONCOKB_URL
+        $PYTHON3_BINARY $README_SCRIPT -o $ONCOKB_ANNOTATOR_README_FILE
         if [ $? -ne 0 ] ; then
             echo "Failed to generate README, exiting..."
             ONCOKB_ANNOTATION_SUCCESS=0
+        else
+            
```

Comment:
remove extra line

Reply:
Done. Other unneeded whitespace was cleaned up too, and a function positional argument which was being passed unnecessarily was removed.